.github/workflows/test-sensorconnect.yml

```
@@ -17,19 +17,39 @@ name: Test Sensorconnect
 
 on:
   workflow_call:
+    inputs:
+      test_plc:
+        description: run plc if running in merge_queue
+        default: true
```

actionlint (1.7.11)

[error] input "test_plc" of workflow_call event has the default value "true", but it is also required. if an input is marked as required, its default value will never be used

(events)